schema.sql

```diff
@@ -1,5 +1,3 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(id integer, name varchar(20), date_of_birth date, escape_attempts integer, neutered boolean, weight_Kg decimal);
```

Comment:
Great job @Juanpa8830 🤝

- [ ] [Optional] Kindly set your id column to not null and autoincremented and mark it as a primary_key.